QuizApp/App/Services/SecureStorage.swift

```diff
@@ -21,4 +41,20 @@ final class SecureStorage: SecureStorageProtocol {
         keychainService[Key.accessToken.rawValue] = token
     }
 
+    func getToken() throws -> String {
+        guard let token = try keychainService.get(Key.accessToken.rawValue) else {
+            throw SecureStorageError.errorGettingToken
```

Comment:
Why do you throw an error? No token in keychain is a valid case and you should return nil if not found.